calico_containers/integrations/kubernetes/tests/kube_plugin_test.py

```diff
@@ -0,0 +1,378 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+if sys.version_info.major == 2:
+    import __builtin__ as builtins
+else:
+    import builtins
+import json
+import unittest
+import socket
+from nose.tools import assert_equal, assert_true, assert_false
+from mock import patch, Mock, MagicMock, mock_open
+from subprocess import CalledProcessError
+from calico_containers.integrations.kubernetes import calico_kubernetes
+from pycalico.datastore import IF_PREFIX
+
+class NetworkPluginTest(unittest.TestCase):
+
+    def setUp(self):
+        # Mock out sh when importing calico_kubernetes so it doesn't fail when
```

Comment:
This comment says "when importing", which is what it used to do.  However, we're now mocking out "sh" when creating the NetworkPlugin() object, so this comment should be updated.